packages/application-generic/src/usecases/select-integration/select-integration.usecase.ts

```diff
@@ -66,6 +66,15 @@ export class SelectIntegration {
       active: true,
     };
 
+    if (command.identifier) {
+      query = {
+        _organizationId: command.organizationId,
+        channel: command.channelType,
+        identifier: command.identifier,
+        active: true,
+      };
```

Comment:
Let's specify the `_environment` here too. 

Also, I feel that it would be easier if you do, than having the separate if, otherwise if there are any changes to the above we would have to remember to update also what we have in the if:
```
const query: Partial<IntegrationEntity> & { _organizationId: string } = {
      ...(command.id ? { id: command.id } : {}),
      _organizationId: command.organizationId,
      _environmentId: command.environmentId,
      channel: command.channelType,
      ...(command.providerId ? { providerId: command.providerId } : {}),
      active: true,
      ...(command.identifier ? { identifier: command.identifier } : {}),
    };
```

Reply:
ok, that makes sense :) we should state that in the docs in the future :) 
what about active then, we can also allow inactive ones then?

Reply:
I discussed with Dima, and decided to only allow active ones